cypress/integration/examples/cypress_api.spec.js

```diff
@@ -0,0 +1,219 @@
+/// <reference types="cypress" />
```

Comment:
The provided tests are examples of Cypress API usage, but they don't cover the required task. You should write e2e tests that cover creating, editing, and deleting articles, signing in and up with valid and invalid credentials, following/unfollowing users, and updating user information.